secant/Wrappers/WrappedSDKSynchronizer.swift

```diff
@@ -0,0 +1,552 @@
+//
+//  WrappedSDKSynchronizer.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 13.04.2022.
+//
+
+import Foundation
+import ZcashLightClientKit
+import Combine
+import ComposableArchitecture
+
+enum WrappedSDKSynchronizerState: Equatable {
```

Comment:
I thought we had deleted these wrapped things

Reply:
Agreed, it's strange that the wrapped synchronizer is even here.